andela/AndelaEatsUI
# #161079575 Implement suspend engagement (UI)

#### What does this PR do?
Implement `Suspend` Engagement UI

#### Description of Task to be completed?
Ensure there is a UI for Admin to Suspend Engagements

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/engagements)
- You should see a button with `Suspend`
- Click the button and a modal will be displayed

#### What are the relevant pivotal tracker stories?
- [#161079575](https://www.pivotaltracker.com/n/projects/2178749/stories/161079575)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1159" alt="screen shot 2018-11-15 at 4 07 13 pm" src="https://user-images.githubusercontent.com/21138774/48562017-9d61e100-e8f1-11e8-84d2-c17163d617c8.png">
